src/Tasks/Exec.cs

```diff
@@ -687,7 +697,7 @@ private Encoding BatchFileEncoding()
                 case UseUtf8Always:
                     return s_utf8WithoutBom;
                 case UseUtf8Never:
-                    return EncodingUtilities.CurrentSystemOemEncoding;
```

Comment:
Seems strange to me that now, UseUtf8Never may actually use Utf8.

Reply:
Yeah, with this change the behavior is more `UseSystemCodepageAlways`. Think we should add an alias?

Reply:
@livarcocc Did you have an opinion on this?

Reply:
If it would not be too hard, I am in favor of the alias. Might be less surprising for people reading this code in the future (ourselves). What do you think?

Reply:
Done. Can you take another look?